frontend/app/src/components/dashboard/BlockchainBalanceCardList.vue

```diff
@@ -12,7 +12,7 @@
       <v-list-item-content>
         <div class="d-flex flex-row">
           <span class="grow">
-            {{ capitalize(name) }}
+            {{ name | capitalCase }}
```

Comment:
Filters have been removed in Vue 3 https://v3-migration.vuejs.org/breaking-changes/filters.html

Reply:
sad TT